### ISSUE TITLE
Not compatible with Home Assistant 2024.6.1

### ISSUE THREAD

**Issue by NikNakk** [mitch-dc/volkswagen_we_connect_id#261]
We Connect ID no longer works with the latest Home Assistant Core (2024.6.1). This seems to relate to an incompatible version of the requests library (2.32.3 in Home Assistant vs. 2.31.0 in the WeConnect-Python library)

## Version of the custom_component
0.2.0 or 0.2.2

## Installation method (hacs / manual)
HACS

## Installation method of hass (venv, docker, hassio,...)
HASSIO

## Configuration

Default configuration via UI

## Describe the bug

We Connect ID no longer works with the latest Home Assistant Core (2024.6.1). This seems to relate to an incompatible version of the requests library (2.32.3 in Home Assistant vs. 2.31.0 in the WeConnect-Python library)

Reverting to a backup of 2024.6.0 works fine.

## Debug log

```text

Unable to install package weconnect==0.60.2: ERROR: Cannot install weconnect==0.60.2 because these package versions have conflicting dependencies.
ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts
...
Setup failed for custom integration 'volkswagen_we_connect_id': Requirements for volkswagen_we_connect_id not found ['weconnect==0.60.2'].

```

**Comment by RobMeerwijk** [comment]
Same error here on my HA. Also noticed by https://community.home-assistant.io/t/custom-integration-volkswagen-weconnect-id-europe/380933/316

**Comment by juleztb** [comment]
second this with the same error after updating HA to 2024.6.1

**Comment by nanosonde** [comment]
See this one: https://github.com/tillsteinbach/WeConnect-python/issues/192

**Comment by wgumaa** [comment]
Same issue when I upgraded this morning

Logs show:

Logger: homeassistant.setup
Source: setup.py:269
First occurred: 8:39:02 AM (1 occurrences)
Last logged: 8:39:02 AM

Setup failed for custom integration 'volkswagen_we_connect_id': Requirements for volkswagen_we_connect_id not found: ['weconnect==0.60.2'].

**Comment by wgumaa** [comment]
So is it a python version issue?

**Comment by Tassel8062** [comment]
also getting this issue straight after upgrading to 2024.06.01 on version 0.20

**Comment by markbw999** [comment]
Sorry to pile on but same issue for me. A fix would be very much appreciated. Thanks!

**Comment by N1kless** [comment]
Unfortunately same here...I would also very much appreciate a fix. Thank you for the great work.

**Comment by sven-debug** [comment]
Same here. Integrations stopped working and is not configurable anymore. Waiting for a fix. Would be highly appreciated. If help is needed, reach out. 

Warm regards
Sven

**Comment by 20ashdale** [comment]
Same here, works fine again if I restore a backup with core 5.4

**Comment by JacVid** [comment]
Same problem on my side, and I can confirm it's not a question of VW servers because in parallel another custom integration "Volkswagen WeConnect"  works.

**Comment by pdcastro** [comment]
No use in a flood of “me too” comments here, because the code change is needed first in the [WeConnect-python](https://github.com/tillsteinbach/WeConnect-python) library used by this integration, with a different maintainer. Someone else has already created an issue and a PR in that library’s repo:
* https://github.com/tillsteinbach/WeConnect-python/issues/192
* https://github.com/tillsteinbach/WeConnect-python/pull/189

**Comment by johanvh83** [comment]
Same issue. If you we're able to update your integration, that would be highly appreciated!

**Comment by markbw999** [comment]
Amazing! Thanks so much. I only had to restart HA after the edit for it to start working.

**Comment by BennyOtt** [comment]
Are we currently waiting for the other repo to be updated so that the update can go through here too?

**Comment by dikt1338** [comment]
> Are we currently waiting for the other repo to be updated so that the update can go through here too?

This integration requires weconnect. 
1h ago this library ( https://github.com/tillsteinbach/WeConnect-python/ ) was updated so that their requirements fit to the other libraries currently installed in  home assistant.

I guess this integration should work again?

**Comment by NikNakk** [comment]
Closed by commit 97a8360
